tests/integration/integration_steps/fabric_steps.py

```diff
@@ -160,3 +162,60 @@ def assert_events_data_in_fabric(
 
     # Assert that the two DataFrames are equal
     assert_frame_equal(events_dataframe, events_from_lakehouse, check_dtype=False)
+
+
+def parse_abfss_url(url: str) -> tuple[str, str, str]:
+    parsed_url = urlparse(url)
+
+    container_id = parsed_url.netloc.split("@")[0]
+    account_name = parsed_url.netloc.split("@")[1].split(".")[0]
+    file_path = parsed_url.path
+
+    return container_id, account_name, file_path
+
+
+def get_lakehouse_file_client(
+    abfss_prefix: str,
+    table_name: str,
+    file_name: str,
+    credential: DefaultAzureCredential,
+):
+    workspace_name, account_name, lakehouse_file_path = parse_abfss_url(abfss_prefix)
+    lakehouse_file_path = lakehouse_file_path + "/" + table_name + "/"
+
+    service_client = DataLakeServiceClient(
+        f"https://{account_name}.dfs.fabric.microsoft.com", credential=credential
+    )
+    file_system_client = service_client.get_file_system_client(workspace_name)
```

Comment:
Could these calls fail if the config is bad?

Reply:
Added a check with a detailed error message on expected format